docfx_project/ziti/quickstarts/network/aws-quickstart.md

```diff
@@ -50,9 +50,10 @@ the public internet on ports: 22, 443, 1280, 3022.
   * port 1280 - the preselected port the Ziti controller will serve its API over
   * port 3022 - the preselected port for data channels to the Ziti Edge Router
 * Public IP/DNS: Make sure the EC2 instance is assigned a public IP and a public DNS entry
+* You will login via SSH with username *nfadmin* unless you override that default with the EC2 launcher
```

Comment:
i'd rather state that one should not change the default. i'm unsure if anything we did needs that username. it'd be better to just be prescriptive here - "use nfadmin and do not change it" (something along those lines)